templates/views/index.pug

```diff
@@ -80,8 +80,8 @@ block content
                 p Unleash the most powerful weapon in the galaxy, the Supreme Commander, in one of four diverse factions: The United Earth Federation, The Cybran Nation, The Aeon Illuminate or The Order (Seraphim).
         .row.text-center.buy-now
             hr
-            h3 In order to play Forged Alliance Forever, you must first install Supreme Commander: Forged Alliance.  You can purchase it for less than 10€, below:
-            .col-md-12
+            h3 To play Forged Alliance Forever, you must first install Supreme Commander: Forged Alliance on Steam.  You can purchase it for less than 10€, below:
```

Comment:
Please leave out steam as we intend to implement gog linking and then we'll forget to touch it again

Reply:
As soon as you finish the PR started? :stuck_out_tongue_closed_eyes: